docs/how-to-guides/working-with-containers.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Working with Containers
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+`nerdctl` is a Docker-compatible CLI for containerd. The primary goal of `nerdctl` is to facilitate experimenting with cutting-edge features of containerd that are not present in Docker.
+
+[Moby](https://github.com/moby/moby) is an open-source project that was created by Docker to enable and accelerate software containerization. Components include container build tools, a container registry, orchestration tools, and a runtime, and more. The Docker CLI uses the Moby runtime. 
+
+## Running Containers
+
+To run a container with the default `bridge` CNI network (10.4.0.0/24):
+
+<Tabs groupId="container-runtime">
+  <TabItem value="nerdctl" default>
+
+```
+nerdctl run -it --rm alpine
+```
+
+  </TabItem>
+  <TabItem value="docker" default>
+
+```
+docker run -it --rm alpine
+```
+  </TabItem>
+</Tabs>
+
+To build an image using BuildKit:
+
+<Tabs groupId="container-runtime">
+  <TabItem value="nerdctl" default>
+
+```
+nerdctl build -t foo /some-dockerfile-directory
+nerdctl run -it --rm foo
+```
+
+  </TabItem>
+  <TabItem value="docker" default>
+
+  ```
+docker build -t foo /some-dockerfile-directory
+docker run -it --rm foo
+```
+  </TabItem>
+</Tabs>
+
+To build and send output to a local directory using BuiltKit:
+
+<Tabs groupId="container-runtime">
+  <TabItem value="nerdctl" default>
+
+```
+nerdctl build -o type=local,dest=. /some-dockerfile-directory
+```
+  </TabItem>
+  <TabItem value="docker" default>
+
+```
+docker build -o type=local,dest=. /some-dockerfile-directory
+```
+  </TabItem>
+</Tabs>
+
+To run containers from `docker-compose-yaml`:
```

Comment:
This part is duplication of what's under `Docker Compose` section and hence can be removed.

Reply:
Removed as suggested.